README.md

```diff
@@ -3,18 +3,15 @@
 
 TODO: Description.
 
-## Installation
+## Configuration
 
-Download the latest version of the sensu-CHANGEME from [releases][1],
-or create an executable script from this source.
+### Asset Registration
 
-From the local path of the sensu-CHANGEME repository:
+Assets are the best way to make use of this plugin. If you're not using an asset, please consider doing so! If you're using sensuctl 5.13 or later, you can use the following command to add the asset: 
 
-```
-go build -o /usr/local/bin/sensu-CHANGEME main.go
-```
+`sensuctl asset add CHANGEME/sensu-CHANGEME:VERSION`
 
-## Configuration
+If you're using an earlier version of sensuctl, you can find the asset on the [Bonsai Asset Index](https://bonsai.sensu.io/assets/CHANGEME/sensu-CHANGEME).
```

Comment:
https://bonsai.sensu.io/assets/CHANGEME/sensu-CHANGEME 

should ^ be https://bonsai.sensu.io/ instead?

Reply:
@hillaryfraley this should link directly to the asset, rather than just Bonsai in general. Better UX to send folks straight to what they need vs them having to search for it.

Reply:
Got it!